apps/properties/README.md

```diff
@@ -1,25 +1,38 @@
 # Properties
 
-Asynchronously check test outcomes that are expected to
-validate but may not immediately.
+Properties provides a series of macros for accessing
+application configuration (properties) within your modules.
 
-AssertAsync implements a macro that injects the necessary
-retry logic for validating test assertions with a configurable
-delay between checks and maximum number of attempted checks.
+Properties is written  to require that each module has its
+own configuration block in the umbrella top-level `releases.exs`
+file. This ensures that runtime configuration is testable.
+
+It exposes two macros, `get_config_value/2` which gets the
+value assigned to the configuration key as well as `getter/2`
+which creates a getter function, named for the supplied key,
+you can call within the body of the module to return the
+current value assigned to that key.
+
+Both macros allow you to specify default values and whether
+or not the requested key should be required and handles either
+scenario accordingly.
 
 ## Usage
 
 ```elixir
   defmodule Example do
-    use ExUnit.Case
-    import AssertAsync
+    use Properties, otp_app: :example_service
+
+    @app_name get_config_value(:app_name, required: true)
```

Comment:
Should we be clear about not using this format unless absolutely necessary?

Reply:
which format specifically? using get_config_value and binding it to a module attribute?

Reply:
yeah, the goal is to make it painless to access the config at runtime and get_config_value should only be used when the config has to be done a compile time.

Reply:
sure thing.